src/app/settings/views/Configuration/GeneralForm/ThemedRadioButton/_index.scss

```diff
@@ -61,12 +61,12 @@
 
         .general-form__radio::before {
             content: "";
-            width: 8px;
-            height: 8px;
+            width: 0.5rem;
+            height: 0.5rem;
             border-radius: 50%;
             transform: scale(0);
             transition: 120ms transform ease-in-out;
-            box-shadow: inset 8px 8px #fff;
+            box-shadow: inset 0.52rem 0.52rem #fff;
```

Comment:
Would it be worth leaving a comment here about the specific size making it work when zoomed?

Reply:
I'll do just that